trino/client.py

```diff
@@ -112,18 +112,21 @@ class ClientSession:
     def __init__(
         self,
         user: str,
-        authorization_user: str = None,
-        catalog: str = None,
-        schema: str = None,
-        source: str = None,
-        properties: Dict[str, str] = None,
-        headers: Dict[str, str] = None,
-        transaction_id: str = None,
-        extra_credential: List[Tuple[str, str]] = None,
-        client_tags: List[str] = None,
-        roles: Union[Dict[str, str], str] = None,
-        timezone: str = None,
+        authorization_user: Optional[str] = None,
+        catalog: Optional[str] = None,
+        schema: Optional[str] = None,
+        source: Optional[str] = None,
+        properties: Optional[Dict[str, str]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        transaction_id: Optional[str] = None,
+        extra_credential: Optional[List[Tuple[str, str]]] = None,
+        client_tags: Optional[List[str]] = None,
+        roles: Optional[Union[Dict[str, str], str]] = None,
+        timezone: Optional[str] = None,
     ):
+        self._object_lock = threading.Lock()
+        self._prepared_statements: Dict[str, str] = {}
```

Comment:
was the reordering required?

Reply:
No, I moved them since they are not passed as arguments and it’s easier to distinguish logical difference. Can move back if needed.